docs/doxygen/config.doxyfile

```diff
@@ -2226,7 +2226,7 @@ SKIP_FUNCTION_MACROS   = YES
 # the path). If a tag file is not located in the directory in which doxygen is
 # run, you must also specify the path to the tagfile here.
 
-TAGFILES               =
+TAGFILES               = source/dependency/coreMQTT/docs/doxygen/output/mqtt.tag=https://docs.aws.amazon.com/embedded-csdk/202103.00/lib-ref/libraries/standard/coreMQTT/docs/doxygen/output/html
```

Comment:
Would be better to link to the doxygen hosted on freertos.org
```suggestion
TAGFILES               = source/dependency/coreMQTT/docs/doxygen/output/mqtt.tag=https://freertos.org/Documentation/api-ref/coreMQTT/docs/doxygen/output/html`

Reply:
Sure